src/i18n/locales/en/translation.json

```diff
@@ -151,11 +151,11 @@
   },
   "send": {
     "loading": "Loading",
-    "title": "Send bitcoin",
-    "subtitle": "Collaborative transactions increase the privacy of yourself and others.",
+    "title": "Send",
+    "subtitle": "Use a collaborative transaction for increased security and privacy.",
```

Comment:
What's the security aspect to it?

Reply:
Two sides of the same coin, see [Lopp's List](https://github.com/jlopp/physical-bitcoin-attacks).

There is a reason why the *s* in `https` stands for "secure" even though it is technically "private" - I think we should slowly but surely get this into the heads of bitcoiners too.

Reply:
Fair point! 👍